README.md

```diff
@@ -261,6 +260,39 @@ ENV BROKER_TOKEN      secret-broker-token
 ENV ARTIFACTORY_URL   <yourdomain>.artifactory.com
 ```
 
+### Nexus 3
+
+To use the Nexus 3 client with an Nexus 3 deployment, run `docker pull snyk/broker:nexus` tag. The following environment variables are needed to customize the Broker client:
+
+- `BROKER_TOKEN` - the snyk broker token, obtained from your artifactory integration settings view.
+- `BASE_NEXUS_URL` - the URL of your Nexus 3 deployment, such as `https://[<user>:<pass>@]<your.nexus.hostname>`.
+- `BROKER_CLIENT_VALIDATION_URL` - Nexus validation url, checked by broker client systemcheck endpoint. If Nexus user requires auth, use `$BASE_NEXUS_URL/service/rest/v1/status/check` (e.g. `https://<user>:<pass>@<your.nexus.hostname>/service/rest/v1/status/check`) otherwise use `$BASE_NEXUS_URL/service/rest/v1/status` (e.g. `https://<your.nexus.hostname>/service/rest/v1/status`).
+
+#### Command-line arguments
+
+You can run the docker container by providing the relevant configuration:
+
+```console
+docker run --restart=always \
+           -p 7341:7341 \
+           -e BROKER_TOKEN=secret-broker-token \
+           -e BASE_NEXUS_URL=https://[<user>:<pass>@]<your.nexus.hostname> \
+           -e BROKER_CLIENT_VALIDATION_URL=https://<your.nexus.hostname>/service/rest/v1/status[/check] \
```

Comment:
should we mention RES_BODY_SUB_URL? (only for npm/yarn)

Reply:
^ I read the comments. I see what you mean. I will update.